### PR TITLE
Test coverage for unary

### PR DESCRIPTION
**Pull request by Axkea**

### CODE REVIEW

File: protocol/triple/triple_protocol/client_ext_test.go [hunk 1]
```diff
@@ -1,181 +1,156 @@
...
+		ctx := context.Background()
+		// unary
+		var message *pingv1.PingRequest
+		err := client.Ping(ctx, triple.NewRequest(message), triple.NewResponse(&pingv1.PingResponse{}))
+		assert.Nil(t, err)
+		text := strings.Repeat(".", 256)
+		msg := &pingv1.PingResponse{}
+		err = client.Ping(ctx, triple.NewRequest(&pingv1.PingRequest{Text: text}), triple.NewResponse(msg))
+		assert.Nil(t, err)
+		assert.Equal(t, msg.Text, text)
```
[the remark this answers is not in the document]
Comment (Axkea) [reply protocol/triple/triple_protocol/client_ext_test.go:78]: 流的那一块还有点问题，所以把流的测试注释掉了

File: protocol/triple/triple_protocol/client_ext_test.go [hunk 2]
```diff
@@ -1,181 +1,156 @@
...
-//	}
-//}
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package triple_protocol_test
+
+import (
```
[the remark this answers is not in the document]
Comment (Axkea) [reply protocol/triple/triple_protocol/client_ext_test.go:16]: 好的

**Review by chickenlj — APPROVED** [review]
LGTM.

**Comment by chickenlj** [comment]
Please run the following command to format import.

```sh
go get -u github.com/dubbogo/tools/cmd/imports-formatter

imports-formatter
```

**Pull request merged**